docs/markdown/pipelines/exomeseq.md

```diff
@@ -1,38 +1,54 @@
 # WES & WGS analysis
 
-To perform DNA seq and Variant calling analyses, we use the [nf-core/sarek](https://github.com/nf-core/sarek) pipeline.
-The pipeline can be utilized for Whole genome sequencing (WGS) and Whole exome sequencing (WES) analyses.
+To perform DNA seq and Variant calling analyses for short-read normal, tumor-only, and paired samples, we use the [nf-core/sarek](https://github.com/nf-core/sarek) pipeline.
+Initially designed for Human, and Mouse, it can work on any species with a reference genome. Sarek can also handle tumour / normal pairs and could include additional relapses.
```

Comment:
```suggestion
Initially designed for Human, and Mouse, it can work on any species with a reference genome. Sarek can also handle tumor / normal pairs and could include additional relapses.
```

Reply:
American spelling all the way :D